metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/sql/DirectSqlGetPartition.java

```diff
@@ -74,6 +76,12 @@
 public class DirectSqlGetPartition {
     private static final String FIELD_DATE_CREATED = "dateCreated";
     private static final String FIELD_BATCHID = "batchid";
+    private static final String WAP_TABLE_DB = "audit";
```

Comment:
Lets not use the abbreviation WAP. Lets just use AUDIT.Ex: AUDIT_DB

Reply:
this sounds like it should be a property somehow or a feature you can turn on and off. Not everyone might have this?